std_msgs/Header header
mocap_ros/Detection[] detections
